### PR TITLE
Message describing the changes here

### PR DESCRIPTION
**Pull request by PatriziaBussatori**
I changed the description and made other examples

### CODE REVIEW

**Comment by PatriziaBussatori** [comment]
Good work

File: participants/Bussatori/doc/README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+Hi to everyone! This is a small guide to use my python scripts for didactic in astronomy department. Here there are some useful functions for Mathematics classes. Here tehre are some examples:
```
Comment (NicoGalvarino) [inline participants/Bussatori/doc/README.md]: Nice and warm readme!

Comment (NicoGalvarino) [reply participants/Bussatori/doc/README.md]:
```suggestion
Hi to everyone! This is a small guide to use my python scripts for didactic in astronomy department. Here there are some useful functions for Mathematics classes. Here are some examples:
```

**Comment by ceb8** [comment]
Thanks! 😄

**Pull request merged**